pkg/asset/manifests/node.go

```diff
@@ -0,0 +1,82 @@
+package manifests
+
+import (
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"path/filepath"
```

Comment:
nit: please move this into its own import block and then delete the following blank line.

import blocks should follow:

```
<stdlib>

<external>

<openshift>
```

Reply:
Addressed